zigzag/zigzag_test_log.py

```diff
@@ -973,3 +1057,14 @@ def _parse_test_cases_without_steps(self):
 
         for testcase_xml in self._mediator.junit_xml.findall(testcase_xpath):
             self._test_logs.append(_ZigZagTestLog(testcase_xml, self._mediator))
+
+    def _parse_tempest_test_cases(self):
+        """Parse JUnitXML generated by tempest"""
+
+        for testcase_xml in self._mediator.junit_xml.findall('testcase'):
+            self._test_logs.append(_ZigZagTestLogTempest(testcase_xml, self._mediator))
+
+
+class ZigZagTestLogError(Exception):
+    """An Error used by _ZigZagTestLog"""
+    pass  # we are using this to raise in the event we cant find the automation content
```

Comment:
Add a proper initializer to capture the message and assign it to whatever the PEP8 best practice is for storing an exception message.